dusty/payload.py

```diff
@@ -13,6 +15,8 @@ def __eq__(self, other):
         return False
 
     def run(self):
+        if self.client_version != VERSION:
+            raise RuntimeError("Dusty daemon is running version: {}, and client is running version: {}".format(VERSION, self.client_version))
```

Comment:
this is not the method the daemon uses to actually run the Payload.  This is just a helper function, which is actually just used by the client ATM.  Check the daemon code to see how the daemon should check the client_version